eng/common/pipelines/templates/steps/install-powershellyaml.yml

```diff
@@ -0,0 +1,7 @@
+steps:
+  - task: NuGetToolInstaller@1
+    displayName: 'Use NuGet '
+
+  - powershell: |
+      nuget install powershell-yaml -Version 0.4.2 -OutputDirectory $(Build.SourcesDirectory)
```

Comment:
Why not call install module and passing it a particular PSRegistry?

Reply:
That does not work with the Azure DevOps feed for some reason.

Reply:
Interesting. I assume you correctly registered the feed with PS before trying to use it.

Reply:
https://docs.microsoft.com/en-us/azure/devops/artifacts/tutorials/private-powershell-library?view=azure-devops#connecting-to-the-feed-as-a-powershell-repo

Reply:
Ok I see, I knew it was a authentication problem but I wanted to see if there was a way I could pull from that feed without needing to authenticate again

Reply:
I don't believe you need to authenticate to our public feeds, at least I would hope not.

Reply:
Well the instructions you sent me requires PAT. Or am I missing something. When I tried it locally I was able to ass the Package source but could not pull the module form it.

Reply:
authentication is only for private feeds.
```
Register-PSRepository -Name azure-sdk-tools-feed -SourceLocation "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-tools/nuget/v2" -PublishLocation "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-tools/nuget/v2" -InstallationPolicy Trusted
Find-Module -Repository azure-sdk-tools-feed
Install-Module -Repository azure-sdk-tools-feed powershell-yaml
```
Those steps appear to work for me locally.

Reply:
Oh I see. I was using the wrong SourceLocation